en/lesson-08.md

```diff
@@ -16,18 +16,18 @@ following in your preamble and we're good to go:
 
 In order to typeset a `tabular` we have to tell LaTeX how many columns will be
 needed and how they should be aligned. This is done in a mandatory argument
-&ndash; often referred to as table preamble &ndash; to the `tabular`
-environment, in which you specify the columns by using single letter names. The
+&mdash;often referred to as 'table preamble'&mdash; to the `tabular`
+environment, in which you specify the columns by using single letter-names. The
```

Comment:
If there should be a hyphen, I think it should be "single-letter names", as those are column names using a single letter and not the names of some letters.

Reply:
I don't know what I was thinking about when I wrote that. Probably leaning my ABCs :-)